### PR TITLE
Fix issue with preflight and decorator plugin

### LINKED ISSUE #188: "Unexpected situation arose" error with transform-decorators-legacy plugin
Repo: https://github.com/arackaf/booklist/tree/feature/rollup  (the rollup branch specifically).

In the `react-redux` folder, `npm i`, then do `npm run rollup`.  You should get an error of

> An unexpected situation arose. Please raise an issue at https://github.com/rollup/rollup-plugin-babel/issues. Thanks!

Removing the `"transform-decorators-legacy"` babel plugin causes the error to go away (and nothing else seems to).

### PR DESCRIPTION
**Pull request by drewpc**
Similar to #190, but fixes #188 for the 3.x branch.

### CODE REVIEW

**Comment by Andarist** [comment]
thanks for the PR, could u add a test case for this?

File: test/test.js [hunk 1]
```diff
@@ -131,6 +131,16 @@ describe( 'rollup-plugin-babel', function () {
 		});
 	});
 
+	it( 'works with bablerc: false and transform-decorators-legacy (#188)', function () {
+		return rollup.rollup({
+			entry: 'samples/transform-decorators-legacy/main.js',
+			plugins: [ babelPlugin({
+				babelrc: false,
```
Comment (drewpc) [inline test/test.js]: @Andarist - I noticed that I had to put `babelrc: false` here an list my specific plugins in the test case.  If I left out `babelrc: false`, the preflight check would not find the `.babelrc` file in the sample directory.  I think this also impacts the test case on lines 127-132 (and likely other test cases).

File: test/test.js [hunk 2]
```diff
@@ -131,6 +131,16 @@ describe( 'rollup-plugin-babel', function () {
 		});
 	});
 
+	it( 'works with bablerc: false and transform-decorators-legacy (#188)', function () {
```
Comment (Andarist) [inline test/test.js]: `bablerc` -> `babelrc`

File: test/test.js (hunk 1 above)
Comment (Andarist) [reply test/test.js]: Interesting, could u create an issue linking to this comment? This needs an investigation

**Comment by Andarist** [comment]
Looks good, do u know if `master` is affected by this too?

**Comment by drewpc** [comment]
@Andarist - will you accept this fix for the 3.x branch now?

**Comment by Andarist** [comment]
I've merged in your change, but it seems that I do not have publish rights atm.

**Comment by Andarist** [comment]
@drewpc I've released new version

**Pull request merged**